java-monitoring/run_protoc.sh

```diff
@@ -0,0 +1,798 @@
+#!/usr/bin/env bash
+
+mkdir -p sandbox/gapic_generator_java/rules_java_gapic
+mkdir -p sandbox/com_google_protobuf/descriptor_proto
+mkdir -p sandbox/com_google_protobuf/duration_proto
+mkdir -p sandbox/com_google_protobuf/any_proto
+mkdir -p sandbox/com_google_protobuf/timestamp_proto
+mkdir -p sandbox/com_google_protobuf/struct_proto
+mkdir -p sandbox/com_google_protobuf/empty_proto
+mkdir -p sandbox/com_google_protobuf/field_mask_proto
+mkdir -p sandbox/com_google_protobuf/wrappers_proto
+mkdir -p sandbox/google/api
+mkdir -p sandbox/google/rpc
+mkdir -p sandbox/google/type
+mkdir -p sandbox/google/monitoring/v3
+
+# Section 0: prepare toolings
```

Comment:
I assume curl commands to archives in GitHub?

Reply:
For `googleapis`, yes.

I still need to investigate how to generate runfiles for `gapic-generator-java`.

Reply:
Assuming you remember its location, why can’t curl get the file?

Reply:
By the way I'm not sure curl can get googleapis. Git can.

Reply:
sorry I mean `git clone` for googleapis and protobuf.